packages/animations/example/lib/fade_scale_transition.dart

```diff
@@ -67,7 +67,6 @@ class _FadeScaleTransitionDemoState extends State<FadeScaleTransitionDemo>
                   onPressed: () {
                     showModal<void>(
                       context: context,
-                      configuration: const FadeScaleTransitionConfiguration(),
```

Comment:
I think we can just leave this here, but it's up to you if this becomes the default behavior

Reply:
Thanks, I think the sample code should be simple, so omitting default value is better 🤔